networkx/classes/graph.py

```diff
@@ -1993,7 +1993,7 @@ def nbunch_iter(self, nbunch=None):
         """Returns an iterator over nodes contained in nbunch that are
         also in the graph.
 
-        The nodes in nbunch are checked for membership in the graph
+        The nodes in iterable nbunch are checked for membership in the graph
```

Comment:
`nbunch` is not an iterable. It can be an iterable. But it can also be a node or it can be `None`.

Reply:
Nodes from nbunch that are integers that aren't in self are not silently ignored but get raised an error, so there was a need to reflect the change in the documentation.

None is used as the edge case for all nodes.

The "silently ignore" part was only applicable to iterable nbunches, so I tried to change the documentation accordingly. Please give me additional feedback.

Reply:
Got it!  I see (now) that the original docs were also talking about nbunch as an iterable in this sentence.
So adding the adjective "iterable" makes sense. I think it is even slightly better to add "an iterable" here... but it is ok as is too...
```suggestion
        The nodes in an iterable nbunch are checked for membership in the graph
```

Reply:
I agree that it's fine as is, but I wanted to change it mainly because I got confused after reading it.